app/src/main/java/ch/epfl/sweng/erpa/model/UuidObject.java

```diff
@@ -0,0 +1,7 @@
+package ch.epfl.sweng.erpa.model;
```

Comment:
What's the difference with the UidObject interface?

Reply:
You mean UUID, uid should not exist anywhere

Reply:
I asked when it still existed. If it's gone, my comment shouldn't stand anymore :)